File: nos2jdbc-tutorial-spring/src/main/java/nos2jdbc/tutorial/spring/entity/LunchFee.java
package nos2jdbc.tutorial.spring.entity;

import java.math.BigDecimal;
import java.time.LocalDate;

import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;

@Entity
public class LunchFee extends EntityBase {
    public Long memberId;
    @ManyToOne
    public Member member;
    
    public LocalDate payOn;
    public BigDecimal amount;
}

File: nos2jdbc-tutorial-kotlin-spring/src/main/kotlin/nos2jdbc/tutorial/kotlinspring/entity/LunchFee.kt
package nos2jdbc.tutorial.kotlinspring.entity

import java.math.BigDecimal
import java.time.LocalDate

import jakarta.persistence.Entity
import jakarta.persistence.ManyToOne

@Entity
class LunchFee : EntityBase() {
    var memberId: Long? = null
    @ManyToOne
    var member: Member? = null
    
    var payOn: LocalDate? = null
    var amount: BigDecimal? = null
}
